src/entities/beaconSearchResult.ts

```diff
@@ -0,0 +1,42 @@
+export interface IBeaconSearchResult {
```

Comment:
@tm1303 This can be removed as we won't be using the old search endpoint which includes the uses/owners/emergency contacts